.eslintrc.js

```diff
@@ -26,7 +25,10 @@ module.exports = {
       }
     ],
     '@typescript-eslint/camelcase': 'off',
-    '@typescript-eslint/explicit-function-return-type': 'off'
+    '@typescript-eslint/explicit-function-return-type': 'off',
+    'react/prop-types': 'off',
+    '@typescript-eslint/no-empty-function': 'off',
+    '@typescript-eslint/ban-ts-comment': 'off'
```

Comment:
Preferably we wouldn't turn this off but rather use `@ts-expect-error` with a description.

Reply:
i changed the rule. but now linting fails. is there someone else more qualified to add description? i can see that thuey added some of them

Reply:
Alternatively to get this over the finish line, we could change it to `warn` instead of off.

Reply:
done